src/main/java/rtg/world/WorldTypeRTG.java

```diff
@@ -80,10 +53,30 @@ public IChunkProvider getChunkGenerator(World world, String generatorOptions)
             return new ChunkProviderGenerate(world, world.getSeed(), world.getWorldInfo().isMapFeaturesEnabled());
         }
     }
-	
+
     @Override
-    public float getCloudHeight()
-    {
+    public float getCloudHeight() {
         return 256F;
     }
+
+    private static Runnable clearChunkManager() {
+        return new Runnable() {
+            public void run() {
+                chunkManager = null;
+            }
+        };
+    }
+
+    private static Runnable clearChunkProvider() {
+        return new Runnable() {
+            public void run() {
+                chunkProvider = null;
+            }
+        };
+    }
+
+    public boolean isWorldTypeRTG() {
```

Comment:
that wass the one i was refering to. It may screw up when the server has more than one world